drivers/include/tcs37727.h

```diff
@@ -40,9 +40,19 @@ extern "C"
 #define TCS37727_I2C_ADDRESS    0x29    /**< Default Device Address */
 #endif
 
+/**
+ * @defgroup drivers_tcs37727_config     TCS37727 RGB Light Sensor driver compile configuration
+ * @ingroup config_sensors
+ * @{
+ */
+
+/**
+ * @brief Set the default RGBC integration time.
```

Comment:
```suggestion
 * @brief Default RGBC integration time.
```
Also, now that we are changing this, can we add the units to the doc?